docs/modules/release-notes/pages/2023/v-4-1-jan-7-2023.adoc

```diff
@@ -0,0 +1,93 @@
+=  v4.1 | January 7, 2023
```

Comment:
Remove the extra space between `=` and `v4.1`.

Before: `=  v4.1 | January 7, 2023`
After: `= v4.1 | January 7, 2023`